Documentation/benchmarks/etcd-2-2-benchmarks.md

```diff
@@ -0,0 +1,69 @@
+## Physical machines
+
+GCE n1-highcpu-2 machine type
+
+- 1x dedicated local SSD mounted under /var/lib/etcd
+- 1x dedicated slow disk for the OS
+- 1.8 GB memory
+- 2x CPUs
+
+## etcd Cluster
+
+3 etcd 2.2.0-alpha.1 members, each runs on a single machine.
+
+Detailed versions:
+
+```
+etcd Version: 2.2.0-alpha.1+git
+Git SHA: 28b61ac
+Go Version: go1.4.2
+Go OS/Arch: linux/amd64
+```
+
+Also, we use 3 etcd 2.1.0 alpha-stage members to form cluster to get base performance. etcd's commit head is at [c7146bd5](https://github.com/coreos/etcd/commits/c7146bd5f2c73716091262edc638401bb8229144), which is the same as the one that we use in [etcd 2.1 benchmark](./etcd-2-1-0-benchmarks.md).
+
+## Testing
+
+Bootstrap another machine and use benchmark tool [boom](https://github.com/rakyll/boom) to send requests to each etcd member. Check [here](../../hack/benchmark/) for instructions.
+
+## Performance
+
+### reading one single key
+
+| key size in bytes | number of clients | target etcd server | read QPS | 90th Percentile Latency (ms) | read QPS (base) | 90th Percentile Latency (ms) (base) |
+|-------------------|-------------------|--------------------|----------|---------------|----|----|
+| 64                | 1                 | leader only        | 2216 (+5%) | 0.5 (-17%) | 2104  | 0.6 |
+| 64                | 64                | leader only        | 16038 (-10%) | 6.1 (+0%) | 17828 | 6.1 |
+| 64                | 256               | leader only        | 15497 (-16%) | 22.4 (+5%) | 18551 | 21.3 |
+| 256               | 1                 | leader only        | 2115  (-8%) | 0.5 (+0%) | 2291  | 0.5 |
+| 256               | 64                | leader only        | 16083 (-13%) | 6.1 (+8%) | 18407 | 5.6 |
+| 256               | 256               | leader only        | 15444 (-17%) | 21.9 (+2%) | 18403 | 21.3 |
+| 64                | 64                | all servers        | 45101 (-9%) | 2.1 (+5%) | 49089 | 2.0 |
+| 64                | 256               | all servers        | 50558 (-14%) | 8.0 (+8%) | 58129 | 7.4 |
+| 256               | 64                | all servers        | 45415 (-8%) | 2.1 (+5%) | 49207 | 2.0 |
+| 256               | 256               | all servers        | 50531 (-14%) | 8.1 (+20%) | 58725 | 6.7 |
```

Comment:
the `read QPS` looks quite narrow, and markdown doesn't support to adjust the width of column. :(